pkg/konnector/controllers/cluster/serviceexport/cluster-scoped/utils.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2023 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterscoped
+
+import (
+	"errors"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func Prepend(name, clusterNs string) string {
+	return clusterNs + "-" + name
+}
+
+func Behead(name, clusterNs string) string {
```

Comment:
is behead a word? Why not Strip or Trim?

Reply:
also go doc please

Reply:
> also go doc please

Done in d9cdc0f.

Reply:
> is behead a word? Why not Strip or Trim?

Yes! `Behead` is a word. I wanted to be as accurate as possible to catch the meaning of 'trim from the head but not tail', just like the function name `Prepend` catches similar meaning. So I looked into dictionaries and (for my first time) learnt this word, which is the closest one I can find.